int/api/cmd/deploySC.go

```diff
@@ -22,7 +27,9 @@ type deploySC struct {
 }
 
 func (d *deploySC) Handle(params operations.CmdDeploySCParams) middleware.Responder {
-	file, err := io.ReadAll(params.SmartContract)
+	_smartContractBytes, err := base64.StdEncoding.DecodeString(params.Body.SmartContract)
+	smartContractReader := bytes.NewReader(_smartContractBytes)
+	smartContractByteCode, err := io.ReadAll(smartContractReader)
```

Comment:
2 things here:
- The first `err` is never checked
- I think it can be simplified as follow (but I didn't test it):
  ```suggestion
  	smartContractByteCode, err := base64.StdEncoding.DecodeString(params.Body.SmartContract)
  ```
  Because `base64.StdEncoding.DecodeString` already returns a `[]byte`, so there is no need to use `bytes.NewReader` and `io.ReadAll`